pkg/admission/cert/writer/certwriter.go

```diff
@@ -141,19 +154,37 @@ type certReadWriter interface {
 	overwrite(webhookName string) (*generator.Artifacts, error)
 }
 
-func validCert(certs *generator.Artifacts) bool {
-	// TODO:
-	// 1) validate the key and the cert are valid pair e.g. call crypto/tls.X509KeyPair()
-	// 2) validate the cert with the CA cert
-	// 3) validate the cert is for a certain DNSName
-	// e.g.
-	// c, err := tls.X509KeyPair(cert, key)
-	// err := c.Verify(options)
+func validCert(certs *generator.Artifacts, dnsName string, validTime time.Duration) (bool, error) {
```

Comment:
The linter is complaining about parameter `validTime` not being changed :/
It is handy to have this parameter for testing purpose.